IndexedDB/bigint_value.htm

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>Values</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="support.js"></script>
+
+<script>
+// BigInt and BigInt wrappers are supported in serialization, per
+// https://github.com/whatwg/html/pull/3480
+// This support allows them to be used as IndexedDB values.
+
+let i = 0;
+function value(value, test) {
+    var t = async_test(document.title + " - " + (i++));
```

Comment:
It would be great if the test name said something about what was being tested in this case; rather than document.title, how about "BigInts as values in IndexedDB" 

This really helps implementers/reviewers when looking a big table of failures: you can tell what the failure was without reading the test.

Can this be passed in to the `value()` function rather than just using numbers, so it's clear what each case does?